components/infra-proxy-service/server/environments.go

```diff
@@ -246,20 +327,14 @@ func (s *Server) GetEnvironmentRecipes(ctx context.Context, req *request.Environ
 }
 
 // fromAPIToListEnvironments a response.Environments from a struct of Environments
-func fromAPIToListEnvironments(al chef.EnvironmentResult) []*response.EnvironmentListItem {
-	cl := make([]*response.EnvironmentListItem, len(al))
-
-	index := 0
-	for c := range al {
+func fromAPIToListEnvironments(al EnvironmentResult) []*response.EnvironmentListItem {
```

Comment:
As this is also common across, data-bag, node, environment, policy, Can we reuse this function?